src/Services/OrganizationService.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace OpenEMR\Services;
+
+use OpenEMR\FHIR\R4\FHIRDomainResource\FHIROrganization;
+use OpenEMR\FHIR\R4\FHIRElement\FHIRAddress;
+use OpenEMR\FHIR\R4\FHIRElement\FHIRCodeableConcept;
+use OpenEMR\FHIR\R4\FHIRElement\FHIRCoding;
+use OpenEMR\FHIR\R4\FHIRElement\FHIRId;
+use OpenEMR\FHIR\R4\FHIRElement\FHIRMeta;
+use OpenEMR\FHIR\R4\FHIRElement\FHIRReference;
+use OpenEMR\FHIR\R4\FHIRResource\FHIRBundle;
+use OpenEMR\FHIR\R4\FHIRResource\FHIRBundle\FHIRBundleLink;
+use OpenEMR\FHIR\R4\PHPFHIRResponseParser;
+
+class OrganizationService
+{
+    
+    private $id;
+    
+    public function __construct()
+    {
+    }
+    
+    public function setId($id)
+    {
+        $this->id = $id;
+    }
+    
+    public function getAll()
+    {
+        
+        $pharmacySQL = "SELECT pharmacies.id,
+                        name,
+                        email,
+                        city,
+                        state,
+                        zip as postal_code,
+                        country,
+                        line1,
+                        line2
+                    FROM pharmacies left
+                    JOIN addresses ON pharmacies.id = addresses.id; ";
+        
```

Comment:
Unsure why you picked pharmacies because i'd think our default would be facilities.
If client needed pharmacies then we expect to see something like a route of
https://domain/Organization/Pharmacy etc...
The main organization for a practice would be facility. I think including pharmacies could confuse a client.